src/deadline/client/ui/widgets/shared_job_timeout_settings_widget.py

```diff
@@ -0,0 +1,192 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+A UI Widget containing the timeout settings widget.
+"""
+
+from __future__ import annotations
+
+from typing import Any
+
+from qtpy.QtWidgets import (  # type: ignore
+    QFormLayout,
+    QGroupBox,
+    QLabel,
+    QMessageBox,
+    QSpinBox,
+    QCheckBox,
+    QGridLayout,
+    QAbstractSpinBox,
+)
+
+
+SECONDS_IN_A_MINUTE = 60
+SECONDS_IN_AN_HOUR = 60 * 60
+SECONDS_IN_A_DAY = SECONDS_IN_AN_HOUR * 24
+
+
+class SharedJobTimeoutSettingsWidget(QGroupBox):  # pylint: disable=too-few-public-methods
```

Comment:
```suggestion
class JobTimeoutSettingsWidget(QGroupBox):  # pylint: disable=too-few-public-methods
```

nit, I would removed `shared` from the class name and path

Reply:
Oh ok, I thought we needed it because we used in the shared settings tab. Will remove it.